src/CoreBundle/private/library/classes/TCMSTableEditor/TCMSTableEditorEndPoint.class.php

```diff
@@ -1779,15 +1779,13 @@ protected function _WriteDataToDatabase(&$oFields, &$oData, $bDataFromDatabase =
 
         if ($databaseChanged) {
             $this->LoadDataFromDatabase();
-            if (true === $bIsUpdateCall && true === $this->isRecordingActive() && \count($dataForChangeRecorder) > 0) {
-                if (\count($dataForChangeRecorder) > 0) {
-                    $this->writePostWriteLogChangeData(
-                        $bIsUpdateCall,
-                        $dataForChangeRecorder,
-                        $whereConditions,
-                        $setLanguageFields
-                    );
-                }
+            if (true === $this->isRecordingActive() && \count($dataForChangeRecorder) > 0) {
```

Comment:
Very minor detail here: Wouldn't it make more sense to swap the conditions since `count` is the less expensive operation?